src/lib/storage/storage_manager.cpp

```diff
@@ -505,9 +504,9 @@ FILE_HEADER StorageManager::_read_file_header(const std::string& filename) const
   auto file_header = FILE_HEADER{};
   auto fd = int32_t{};
 
-  Assert((fd = open((_persistence_directory + filename).c_str(), O_RDONLY)) >= 0, "Open error");
-  auto* persisted_header =
-      reinterpret_cast<uint32_t*>(mmap(NULL, _file_header_bytes, PROT_READ, MAP_PRIVATE, fd, off_t{0}));
+  Assert((fd = open(filename.c_str(), O_RDONLY)) >= 0, "Open error");
+  auto* map_pointer = mmap(NULL, _file_header_bytes, PROT_READ, MAP_PRIVATE, fd, off_t{0});
+  auto* persisted_header = reinterpret_cast<uint32_t*>(map_pointer);
```

Comment:
Why do we have a change here? It seems to me, that this should not have been changed.

Reply:
Thank you for the comment - yep I think this is an artefact of the rebase.